app/src/main/java/com/tvtracker/MainActivity.kt

```diff
@@ -237,9 +245,12 @@ class MainActivity : ComponentActivity() {
                     Column(modifier = Modifier
                         .padding(8.dp)
                         .fillMaxWidth()
-                        .wrapContentSize(Alignment.Center), horizontalAlignment = Alignment.CenterHorizontally) {
+                        .wrapContentSize(Alignment.Center),
+                        horizontalAlignment = Alignment.CenterHorizontally) {
                         //Title
-                        Text(text = mediaItem.title, fontSize = 24.sp, fontWeight = FontWeight.Bold, textAlign = TextAlign.Center)
+                        Text(text = mediaItem.title, fontSize = 24.sp,
+                            fontWeight = FontWeight.Bold,
+                            textAlign = TextAlign.Center)
```

Comment:
Broke up UI properties for added readability to display different elements implementations, as well as to wrap lines that were too long.
https://developer.android.com/kotlin/style-guide#where_to_break